core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -5943,6 +5999,30 @@ protected SqlNode visitScoped(SqlCall call) {
     }
   }
 
+  /**
+   * Converts an expression into canonical form by fully-qualifying any
+   * identifiers. For common columns in USING, it will be converted to
+   * COALESCE(A.col, B.col) AS col.
+   */
+  static class SelectExpander extends Expander {
+    final SqlSelect select;
+
+    SelectExpander(SqlValidatorImpl validator, SelectScope scope,
+                   SqlSelect select) {
```

Comment:
Seems indent here is not consistent with others.

Reply:
Will do.